<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RootParallelGui</class>
 <widget class="QMainWindow" name="RootParallelGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Root Parallel Gui</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/RootParallelGui.qrc">
    <normaloff>:/RootParallelGui/root-logo.png</normaloff>:/RootParallelGui/root-logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="ModulesTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Index</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/RootParallelGui/root-banner.png&quot; /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Linux Biolinum Keyboard O';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Linux Biolinum Keyboard O';&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Linux Biolinum Keyboard O'; font-size:36pt; color:#447078;&quot;&gt;Root &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Linux Biolinum Keyboard O'; font-size:36pt; color:#447078;&quot;&gt;Parallel &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Linux Biolinum Keyboard O'; font-size:36pt; color:#447078;&quot;&gt;Gui&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQuit">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionOuput_Panel"/>
   </widget>
   <addaction name="menuQuit"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>184</height>
    </size>
   </property>
   <property name="windowIcon">
    <iconset resource="../../resources/RootParallelGui.qrc">
     <normaloff>:/RootParallelGui/root-logo.png</normaloff>:/RootParallelGui/root-logo.png</iconset>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>227</width>
          <height>234</height>
         </rect>
        </property>
        <attribute name="label">
         <string>RootMpi</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="MpiLauncherPushButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Launcher</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/RootParallelGui.qrc">
             <normaloff>:/RootParallelGui/rootmpi-exe.png</normaloff>:/RootParallelGui/rootmpi-exe.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>128</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="OuptutDockWidget">
   <property name="windowTitle">
    <string>Output Messges</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QTextEdit" name="OutputTextEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(220, 220, 220);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="toolButton_2">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOuput_Panel">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ouput Panel</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/RootParallelGui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>RootParallelGui</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOuput_Panel</sender>
   <signal>activated()</signal>
   <receiver>OuptutDockWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>OutputTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>OutputTextEdit</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>658</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>651</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>OutputTextEdit</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
